### ISSUE TITLE
Crash on Server Load

### ISSUE THREAD

**Issue by techdude** [DarkPacks/SevTech-Ages#3504]
## Bug Report

Currently whenever I launch my singleplayer sevtech, it will make it to the main screen, but once I launch the world it crashes before it finishes loading.

## Expected Behaviour

I would expect it to load.

## Possible Solution

The crash log says something about chickens. I have a chicken spawner set up (rftools) but I don't know if this is the problem or not.

## Steps to Reproduce (for bugs)

[https://www.youtube.com/watch?v=lywduWIHWik](https://www.youtube.com/watch?v=lywduWIHWik)

## Logs

* Client/Server Log: https://gist.github.com/techdude/8d2ae8513efe9d7cbe75f3d1e35d5eeb
* Crash Log: https://gist.github.com/techdude/31ac146069b4f9586866a05bbc2fcc76

## Client Information

* Modpack Version: 3.0.8
* Java Version: 1.8.0_51
* Launcher Used: Twitch launcher/forge
* Memory Allocated: 12GB
* Server/LAN/Single Player: Single Player mostly, sometimes open to lan
* Optifine Installed: No
* Shaders Enabled: No

## World Information

* Modpack Version world created in: 3.0.8

* Additional Content Installed: None

## Server Information
* Java Version:
* Operating System:
* Hoster/Hosting Solution:
* Sponge (Non-Vanilla Forge) Server:

**Comment by artdude543** [comment]
This will be resolved when 3.1.0 is out.

**Comment by techdude** [comment]
Just adding a few more things since I finally fixed it. The chicken thing was a red herring. It was a redstone torch and some multipart incompatibility thing. I found it in MC edit as shown in the screenshots, replaced it with air, and the server started up fine.
![fixingserver2](https://user-images.githubusercontent.com/1178952/50533211-48b1a880-0ae2-11e9-8592-d291b82d5fa7.png)
![fixingserver](https://user-images.githubusercontent.com/1178952/50533212-48b1a880-0ae2-11e9-986d-c14b24bb340b.png)
